oemof/solph/plumbing.py

```diff
@@ -65,16 +65,20 @@ class _Sequence(UserList):
     """
     def __init__(self, *args, **kwargs):
         self.default = kwargs["default"]
+        self.default_changed = False
         super().__init__(*args)
 
     def __getitem__(self, key):
+        if not self.default_changed:
+            return self.default
```

Comment:
Do I understand this right, that no iterable object but a scalar is returned, if ` not default_changed`? I think, it kind of strange to have a sequence class that does not return a iterable object. My idea was rather to move this to the flow/node classed instead.